.github/workflows/pull_request_full_build.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - master
```

Comment:
Do we need to keep the `master` here?

Reply:
We can keep for the moment since pull request file will be taken from master branch when merging to master.